run_analysis.py

```diff
@@ -194,6 +187,22 @@ def analysis(config):  # {{{
         ocn_modelvsobs(config, 'sss', streamMap=oceanStreamMap,
                        variableMap=oceanVariableMap)
 
+    if checkGenerate(config, analysisName='streamfunctionMOC', mpasCore='ocean',
+                     analysisCategory='regriddedHorizontal'):
+        print ""
+        print "Plotting streamfunction of Meridional Overturning Circulation (MOC)..."
+        from mpas_analysis.ocean.meridional_overturning_circulation import moc_streamfunction
+        moc_streamfunction(config, streamMap=oceanStreamMap,
+                        variableMap=oceanVariableMap)
+
+#    if checkGenerate(config, analysisName='timeSeriesMOC', mpasCore='ocean',
+#                     analysisCategory='timeSeries'):
+#        print ""
+#        print "Plotting Meridional Overturning Circulation (MOC)..."
+#        from mpas_analysis.ocean.meridional_overturning_circulation import moc_postprocess
+#        moc_postprocess(config, streamMap=oceanStreamMap,
+#                        variableMap=oceanVariableMap)
```

Comment:
Do we need this commented code above?  My preference is to remove it from the final merged code.